src/DsaBottomUp.cc

```diff
@@ -138,10 +136,22 @@ bool BottomUpAnalysis::runOnModule(Module &M, GraphMap &graphs) {
   const bool do_sanity_checks = true;
 #endif
 
+  BrunchTimer buTime("BU_AND_LOCAL");
+  BrunchTimer localTime("LOCAL");
+  localTime.pause();
+
   LocalAnalysis la(m_dl, m_tli, m_allocInfo);
+
+  const size_t totalFunctions = M.getFunctionList().size();
+  size_t functionsProcessed = 0;
+
   for (auto it = scc_begin(&m_cg); !it.isAtEnd(); ++it) {
     auto &scc = *it;
 
+    SEA_DSA_BRUNCH_PROGRESS("BU_FUNCTIONS_PROCESSED", functionsProcessed,
```

Comment:
You should use Stats for this instead of creating extra local variables

Reply:
same as above -- it's in seahorn